tbtadm/controller.cpp

```diff
@@ -581,17 +620,15 @@ void tbtadm::Controller::acl()
             {
                 continue;
             }
-            uuids.push_back(readAndTrim(dir.path() / uniqueIDFilename));
+            File authorizedFile(dir.path() / authorizedFilename,
+                                File::Mode::Read);
+            bool authorized = std::stoi(authorizedFile.read());
+            std::string uuid(readAndTrim(dir.path() / uniqueIDFilename));
+            uuids.insert(std::pair<std::string, bool>(uuid, authorized));
```

Comment:
`std::make_pair` instead of `std::pair` and you wouldn't have to explicitly mention the template parameters.